ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-upgrade.yaml

```diff
@@ -107,6 +107,7 @@ objects:
         # aad integration configuration - we dont test aad so populate as dummy
         export AZURE_AAD_CLIENT_ID=$AZURE_CLIENT_ID
         export AZURE_AAD_CLIENT_SECRET=$AZURE_CLIENT_SECRET
+        export AZURE_REGION=eastus
```

Comment:
Sorry for reviewing these one by one but upgrade.sh also does not seem to use the location. Rest lgtm

Reply:
https://github.com/openshift/release/pull/1611